src/test/java/org/sagebionetworks/bridge/models/assessments/AssessmentReferenceTest.java

```diff
@@ -36,55 +37,70 @@ public void equalsVerifier() {
         // should be globally unique for assessement references and is sufficient for 
         // equality. This makes detecting duplicates easier during validation.
         EqualsVerifier.forClass(AssessmentReference.class)
-            .allFieldsShouldBeUsedExcept("resolver", "id", "sharedId").verify();
+            .allFieldsShouldBeUsedExcept("resolver", "id", "originSharedId", "appId").verify();
```

Comment:
.allFieldsShouldBeUsedExcept(...) might be easier than adding fields every time we update this object.

Reply:
Just noting, EqualsVerifier 2.0 includes .withOnlyTheseFields(...), I'll check out what other changes are included in there in case we want to update later.